src/TeleportJoin.sol

```diff
@@ -211,6 +237,20 @@ contract TeleportJoin {
         emit Mint(hashGUID, teleportGUID, amtToTake, maxFeePercentage, operatorFee, msg.sender);
     }
 
+    /**
+    * @dev External authed function that registers the teleport
+    * @param teleportGUID Struct which contains the whole teleport data
+    **/
+    function registerMint(
+        TeleportGUID calldata teleportGUID
+    ) external auth {
+        bytes32 hashGUID = getGUIDHash(teleportGUID);
```

Comment:
Should we move the code common to `registerMint` and `requestMint` to a common `_registerMint()` internal function?